script/tool/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-## 0.11.1
```

Comment:
It turns out I forgot to bump the version when I added this, and the repo tooling doesn't validate the tool's metadata the way it does for packages. It wasn't important to have `fix` be its own version, so I'm just fixing the changelog rather than doing a separate PR to add the missed version bump.